cloud-operator/values.yaml

```diff
@@ -73,3 +73,6 @@ env:
   streamSuccessPeriods:
     connect: "1h"
     auth: "2h"
+
+# Proxy configuration
+httpsProxy: ""
```

Comment:
Moving this under `env` just above would be more consistent with the other env variables?